packages/apputils/src/sanitizer.ts

```diff
@@ -592,7 +592,15 @@ export class Sanitizer implements ISanitizer {
       bdo: ['dir'],
       blockquote: ['cite'],
       br: ['clear'],
-      button: ['accesskey', 'disabled', 'name', 'tabindex', 'type', 'value'],
+      button: [
+        'accesskey',
+        'data-commandlinker-command',
```

Comment:
I think we will also need to add the `data-commandlinker-args` attribute that can be used to pass command arguments.

Reply:
Done.